private-templates-service/client/src/GraphService.tsx

```diff
@@ -1,27 +1,27 @@
-var graph = require('@microsoft/microsoft-graph-client');
+var graph = require("@microsoft/microsoft-graph-client");
 
-function getAuthenticatedClient(accessToken) {
+function getAuthenticatedClient(accessToken: any) {
   // Initialize Graph client
   const client = graph.Client.init({
     // Use the provided access token to authenticate
     // requests
-    authProvider: (done) => {
+    authProvider: (done: any) => {
       done(null, accessToken.accessToken);
     }
   });
 
   return client;
 }
 
-export async function getUserDetails(accessToken) {
+export async function getUserDetails(accessToken: any) {
   console.log(accessToken);
```

Comment:
Avoid logging sensitive data like access tokens (also if you can add a logging framework your life will become nicer later on)